Formula/pdns.rb

```diff
@@ -52,6 +56,28 @@ def install
     system "./configure", *args
 
     system "make", "install"
+    (var/"log/pdns").mkpath
+    (var/"run").mkpath
+  end
+
+  def caveats
+    <<-EOS.undent
+    pdns_server must be run as root to bind to port 53 in OS X. To work around this
+    you can either change the 'local-port' setting in pdns.conf, (probably not
+    what you want to do), or start pdns_server as root.  This can be accomplished
+    with 'sudo brew services start pdns', but it gives pdns_server too much
+    privilege for the liking of any experienced sysadmin.
+
+    pdns_server has two configuration settings that allow it to drop privilege once
+    it has bound to a privileged port: 'setuid' and 'setgid'.  Both require numeric
+    values to function properly in OS X.
+
+    See https://doc.powerdns.com/md/authoritative/settings/ for a documentation on
+    all pdns_server settings.
+
+    See https://gist.github.com/mprzybylski/2b16a0f7e00762a0444612e1b0dcf78e for
+    useful hints on creating separate, unprivileged, for services like pdns_server.
```

Comment:
That sounds reasonable, @MikeMcQuaid, but I don't see an existing, sensible default for an unprivileged account...

```
Abby-Normal:pdns mikep$ dscl . -list /Users UniqueID | sort -n -k 2
nobody                  -2
root                    0
daemon                  1
_uucp                   4
_taskgated              13
_networkd               24
_installassistant       25
_lp                     26
_postfix                27
_scsd                   31
_ces                    32
_mcxalr                 54
_appleevents            55
_geod                   56
_serialnumberd          58
_devdocs                59
_sandbox                60
_mdnsresponder          65
_ard                    67
_www                    70
_eppc                   71
_cvs                    72
_svn                    73
_mysql                  74
_sshd                   75
_qtss                   76
_cyrus                  77
_mailman                78
_appserver              79
jssadmin                80
_clamav                 82
_amavisd                83
_jabber                 84
_appowner               87
_windowserver           88
_spotlight              89
_tokend                 91
_securityagent          92
_calendar               93
_teamsserver            94
_update_sharing         95
_installer              96
_atsserver              97
_ftp                    98
_unknown                99
_softwareupdate         200
_coreaudiod             202
_screensaver            203
_locationd              205
_trustevaluationagent   208
_timezone               210
_lda                    211
_cvmsroot               212
_usbmuxd                213
_dovecot                214
_dpaudio                215
_postgres               216
_krbtgt                 217
_kadmin_admin           218
_kadmin_changepw        219
_devicemgr              220
_webauthserver          221
_netbios                222
_warmd                  224
_dovenull               227
_netstatistics          228
_avbdeviced             229
_krb_krbtgt             230
_krb_kadmin             231
_krb_changepw           232
_krb_kerberos           233
_krb_anonymous          234
_assetcache             235
_coremediaiod           236
_launchservicesd        239
_iconservices           240
_distnote               241
_nsurlsessiond          242
_nsurlstoraged          243
_displaypolicyd         244
_astris                 245
_krbfast                246
_gamecontrollerd        247
_mbsetupuser            248
_ondemand               249
_xserverdocs            251
_wwwproxy               252
_mobileasset            253
_findmydevice           254
_datadetectors          257
_captiveagent           258
_ctkd                   259
_applepay               260
_hidd                   261
```

...and you can't add a new, unprivileged service user without `sudo`.

Do you see a username in the list above that you think would work?  If not, do you have any other suggestions?